api/src/main/java/org/openmrs/api/PatientService.java

```diff
@@ -106,6 +106,17 @@ public interface PatientService extends OpenmrsService {
 	@Authorized( { PrivilegeConstants.GET_PATIENTS })
 	public Patient getPatientByUuid(String uuid) throws APIException;
 	
+	/**
+	 * Get patient by universally unique identifier.
+	 * 
+	 * @param uuid universally unique identifier
+	 * @throws APIException
+	 * <strong>Should</strong> fetch patient with given identifier
+	 * <strong>Should</strong> return null if patient not found with given identifier
+	 */
+	@Authorized( { PrivilegeConstants.GET_PATIENTS })
+	public List<Patient> getPatientByIdentifier(String identifier, boolean matchIdentifierExactly) throws APIException;
```

Comment:
The previous method we used also took a list of identifier types to make the the query more efficient.   It was also named with "Patients" since it returns a list of patients.
```@Authorized( { PrivilegeConstants.GET_PATIENTS })
	public List<Patient> getPatientsByIdentifier(String name, String identifier,
	        List<PatientIdentifierType> identifierTypes, boolean matchIdentifierExactly) throws APIException;```